Build failure with GHC 8

```
> /tmp/stackage-build9$ stack unpack ratel-0.1.3
Unpacked ratel-0.1.3 to /tmp/stackage-build9/ratel-0.1.3/
> /tmp/stackage-build9/ratel-0.1.3$ runghc -clear-package-db -global-package-db -package-db=/home/stackage/work/builds/nightly/pkgdb Setup configure --package-db=clear --package-db=global --package-db=/home/stackage/work/builds/nightly/pkgdb --libdir=/home/stackage/work/builds/nightly/lib --bindir=/home/stackage/work/builds/nightly/bin --datadir=/home/stackage/work/builds/nightly/share --libexecdir=/home/stackage/work/builds/nightly/libexec --sysconfdir=/home/stackage/work/builds/nightly/etc --docdir=/home/stackage/work/builds/nightly/doc/ratel-0.1.3 --htmldir=/home/stackage/work/builds/nightly/doc/ratel-0.1.3 --haddockdir=/home/stackage/work/builds/nightly/doc/ratel-0.1.3 --flags=
Configuring ratel-0.1.3...
> /tmp/stackage-build9/ratel-0.1.3$ runghc -clear-package-db -global-package-db -package-db=/home/stackage/work/builds/nightly/pkgdb Setup build
Building ratel-0.1.3...
Preprocessing library ratel-0.1.3...
[1 of 2] Compiling Paths_ratel      ( dist/build/autogen/Paths_ratel.hs, dist/build/Paths_ratel.o )
[2 of 2] Compiling Ratel            ( library/Ratel.hs, dist/build/Ratel.o )

library/Ratel.hs:16:1: error:
    Failed to load interface for ‘GHC.SrcLoc’
    Use -v to see a list of the files searched for.
```

--- a/library/Ratel.hs
+++ b/library/Ratel.hs
@@ -13,7 +13,6 @@ import qualified Data.Text as Text
 import qualified Data.Typeable as Typeable
 import qualified Data.UUID as UUID
 import qualified Data.Version as Version
-import qualified GHC.SrcLoc as SrcLoc
 import qualified GHC.Stack as Stack
 import qualified Network.HTTP.Client as Client
 import qualified Network.HTTP.Client.TLS as Client
@@ -70,16 +69,16 @@ toTraces :: Stack.CallStack -> [Trace]
 toTraces callStack = map (uncurry toTrace) (Stack.getCallStack callStack)
 
 
-toTrace :: String -> SrcLoc.SrcLoc -> Trace
+toTrace :: String -> Stack.SrcLoc -> Trace
 toTrace function srcLoc = Trace
-    { traceFile = Just (SrcLoc.srcLocFile srcLoc)
+    { traceFile = Just (Stack.srcLocFile srcLoc)
     , traceMethod = Just (List.intercalate "."
-        [ SrcLoc.srcLocModule srcLoc
+        [ Stack.srcLocModule srcLoc
         , function
         ])
     , traceNumber = Just (List.intercalate ":" (map show
-        [ SrcLoc.srcLocStartLine srcLoc
-        , SrcLoc.srcLocStartCol srcLoc
+        [ Stack.srcLocStartLine srcLoc
+        , Stack.srcLocStartCol srcLoc
         ]))
     }
 

--- a/package.yaml
+++ b/package.yaml
@@ -9,14 +9,14 @@ ghc-options: -Wall
 github: tfausak/ratel
 library:
   dependencies:
-  - aeson >=0.8 && <0.12
-  - base >=4.8 && <4.10
+  - aeson ==0.11.*
+  - base ==4.9.*
   - bytestring ==0.10.*
   - case-insensitive ==1.2.*
   - containers ==0.5.*
   - http-client ==0.4.*
   - http-client-tls ==0.2.*
-  - http-types >=0.8 && <0.10
+  - http-types ==0.9.*
   - text ==1.2.*
   - uuid ==1.3.*
   other-modules:

--- a/stack.yaml
+++ b/stack.yaml
@@ -1,3 +1,3 @@
 install-ghc: true
 require-stack-version: ! '>=1.0.4'
-resolver: nightly-2016-03-07
+resolver: nightly-2016-05-26

--- a/test-suite/RatelSpec.hs
+++ b/test-suite/RatelSpec.hs
@@ -17,18 +17,16 @@ spec = describe "Ratel" $ do
                     let expected = Ratel.Error
                             { Ratel.errorBacktrace = Just
                                 [ Ratel.Trace
-                                    { Ratel.traceFile = Just "library/Ratel.hs"
-                                    , Ratel.traceMethod = Just "Ratel.?callStack"
-                                    , Ratel.traceNumber = Just "61:39"
-                                    }
-                                , Ratel.Trace
                                     { Ratel.traceFile = Just "test-suite/RatelSpec.hs"
                                     , Ratel.traceMethod = Just "RatelSpec.toError"
                                     , Ratel.traceNumber = Just "16:34"
                                     }
                                 ]
                             , Ratel.errorClass = Just "SomeException"
-                            , Ratel.errorMessage = Just "something went wrong"
+                            , Ratel.errorMessage = Just "\
+                                \something went wrong\n\
+                                \CallStack (from HasCallStack):\n\
+                                \  error, called at test-suite/RatelSpec.hs:13:26 in main:RatelSpec"
                             , Ratel.errorSource = Nothing
                             , Ratel.errorTags = Nothing
                             }

